src/coins.ts

```diff
@@ -1201,6 +1316,16 @@ export const wrappedTokens: { [ChainId: string]: Token } = {
     logoURI:
       'https://zapper.fi/images/networks/ethereum/0x0000000000000000000000000000000000000000.png',
   },
+  [ChainId.SOL]: {
+    address: 'So11111111111111111111111111111111111111112',
+    symbol: 'SOL',
+    decimals: 9,
+    chainId: ChainId.SOL,
+    coinKey: CoinKey.SOL,
+    name: 'Wrapped SOL',
+    logoURI:
+      'https://assets.coingecko.com/coins/images/4128/small/solana.png?1640133422',
```

Comment:
Do we really need these params `?1640133422` in the end? 🙂

Reply:
haha no

Reply:
@0xksure let's remove them from all the tokens 😄